### DISCUSSION TITLE
Some site are very quick and some slow as hell

### DISCUSSION THREAD

**Discussion by stanthewizzard** [ZoeyVid/NPMplus#934] [Q&A] [answered]
Hello
here is an exemple with airsonic advanced (works flawlessly with npm)

location /{
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_set_header X-Forwarded-Protocol $scheme;
        proxy_set_header X-Forwarded-Host $http_host;
        proxy_set_header Origin '';
        add_header X-Robots-Tag "noindex";
        proxy_buffering off;
        proxy_request_buffering off;
        proxy_max_temp_file_size 0;
        client_max_body_size 0;
        proxy_pass                         http://docker30service.HQC.STARFLEET.DOM8:4080;
        proxy_redirect                     http:// https://;
}

Thanks for help

**Comment by Zoey2936** [comment]
What do you want to tell me?

**Reply by stanthewizzard** [reply]
that if iptable is on ... it slow between dockers

I don't understand why ?
NPM classic doesn't have this behavior

Thanks thanks thanks

**Reply by Zoey2936** [reply]
Sorry but I have no idea about what you are talking

**Reply by stanthewizzard** [reply]
> Sorry but I have no idea about what you are talking

I have photon OS 1 and photon OS 2
with iptable on booth

NPM Plus is on 1 service is on 2
If iptable is running access to a service through NPM Plus is very very slow
if iptable is not running everything is very quick

**Reply by Zoey2936** [reply, marked as answer]
Sorry no idea
